websockify/websocketproxy.py

```diff
@@ -492,6 +494,12 @@ def websockify_init():
     parser.add_option("--ssl-ciphers", action="store",
             help="list of ciphers allowed for connection. For a list of "
             "supported ciphers run `openssl ciphers`")
+    parser.add_option("--unix-listen",
+            dest="listen_sock",
```

Comment:
We've avoided using `dest` for pretty much everything else, so let's try to keep the same pattern for new stuff.

Reply:
Would ```--listen-unix-sock``` / ```listen_unix_sock``` and ```--listen-unix-sock-mode``` / ```listen_unix_sock_mode``` be fine instead? Or remove the ```-sock``` to make it shorter?

Personally, I find the unix part to be more telling because there are many other socket types one can listen to but with unix listen I would generally default to thinking thats a unix socket automatically. Though, maybe that´s just me.

Reply:
Did remove ```dest``` now. Though, the reason I did pick the names I did is because there already is ```--unix-target```, so made some sence to call it ```--unix-listen``` in the CLI option to me. However, the variables in the code for listen all start with listen ```listen_host``` / ```listen_port``` / ```listen_fd``` etc.

For the ```--inetd``` option ```dest``` is not used but it´s rewritten later to another variable.

```
if opts.inetd:
   opts.listen_fd = sys.stdin.fileno()
```

Should I do that or just leave it as is? (with the variables called ```unix_target``` / ```unix_target_mode```)